source/_components/tesla.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "Tesla"
+description: "Instructions for how to integrate Tesla car into Home Assistant."
+date: 2017-08-28 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Hub
+ha_release: 0.53
+ha_iot_class: "Cloud Polling"
+---
+
+
+The `Tesla` platform offers integration with the [Tesla](https://auth.tesla.com/login) cloud service and offers presence detection as well as sensors such as charger state and temperature.
+
+This platform provides next components:
+ - Binary sensors - such as parking and charger connection.
+ - Sensors - such as Battery level, Inside/Outside temperature.
+ - Device tracker - to track location of your car
+ - Lock - Door lock. Allow you to control Tesla's door lock
+ - Climate - HVAC control. Allow you to control (turn on/off, set target tempearature) your Tesla's HVAC system.
```

Comment:
**Spelling & Grammar**:
- `Allow` sounds more like having authorization. Suggestion: `Allow you to control` -> `Enables you to control`
- `tempearature` -> `temperature`